src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -1102,8 +1116,8 @@ void EmitFixedSet_LeftToRight()
                     }
                     else
                     {
-                        // We have an arbitrary set of characters that includes at least one non-ASCII char.  We use a custom IndexOfAny helper that
-                        // will perform the search as efficiently as possible.
+                        // We have an arbitrary set of characters that's really large or otherwise not enumerable.
```

Comment:
> or otherwise not enumerable

Does that mean cases like a Unicode category where the exact set of characters may differ?

Reply:
Yes.  The set representation is basically two parts: ranges and categories, where the former is essentially a list of begin/end character ranges and the latter is essentially a list of UnicodeCategory values.  We bail on trying to enumerate the set contents if there are any categories.  That means, for example, this PR won't change what code is generated for `[\s]`.

Reply:
Makes sense, thanks